addons/turbowarp-player/userscript.js

```diff
@@ -99,4 +99,4 @@ export default async function ({ addon, console, msg }) {
 
     scratchStage = document.querySelector(".guiPlayer");
   }
-}
+}
```

Comment:
Adding back the new line should fix the Prettier checks.

Reply:
Thanks!